homeassistant/components/totalconnect/button.py

```diff
@@ -79,6 +79,11 @@ def press(self) -> None:
         """Press the bypass button."""
         self._zone.bypass()
 
+    async def async_press(self) -> None:
+        """Press the bypass button and refresh state."""
+        await super().async_press()
```

Comment:
```suggestion
        await self.hass.async_add_executor_job(self._zone.bypass)
```
and then remove the `press`